### PR TITLE
Removes voter reg query block, query and content

### PR DESCRIPTION
**Pull request by lindsayrmaher**
### What's this PR do?

This pull request removes the voter registration badge!

### How should this be reviewed?

👀 

### Any background context you want to provide?

We want to be sure we aren't incentivizing members to register to vote/vote (by getting more scholarships because of the number of badges etc). 

### Relevant tickets

References [Pivotal #175416879](https://www.pivotaltracker.com/story/show/175416879).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [x] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: docs/development/features/badges.md [hunk 1]
```diff
@@ -29,7 +29,6 @@ Unless specified, badges will only show up in the Badges tab of the user profile
 -   **2 Staff Faves** - This badge remains hidden until earned. Awarded when a second post gets tagged with `good-submission`.
 -   **3 Staff Faves** - This badge remains hidden until earned. Awarded when a third post gets tagged with `good-submission`.
 -   **News Expert** - Awarded when a user's `email_subscription_topics` contains "news" (which means that they are subscribed to The Breakdown)
--   **Registered Voter** - Awarded when a user's `voter_registration_status` is `CONFIRMED` (they told us they are registered) or `REGISTRATION_COMPLETE` (they registered through us)
```
Comment (aaronschachter) [inline docs/development/features/badges.md:32]: What do you think about moving this to a new section for "Retired Badges"? I think might help for institutional knowledge (wait, didn't we once have a Registered Voter badge? let me search Github to see if we removed that once)

File: resources/assets/components/pages/AccountPage/Badges/BadgesTab.js [hunk 2]
```diff
@@ -35,15 +32,6 @@ const NEWSLETTER_BADGE = gql`
...
 
-const VOTER_BADGE = gql`
-  query VoterRegBadgeQuery($userId: String!) {
-    user(id: $userId) {
-      id
-      voterRegistrationStatus
-    }
-  }
-`;
-
```
Comment (aaronschachter) [inline resources/assets/components/pages/AccountPage/Badges/BadgesTab.js:46]: 🙀  at the DRY that could've been (part 1)

File: resources/assets/components/pages/AccountPage/Badges/BadgesTab.js [hunk 3]
```diff
@@ -125,24 +113,6 @@ const badgeModalContent = {
...
   },
-  voterBadge: {
-    title: 'REGISTERED VOTER',
-    earnedText:
-      'Congratulations! You just took a step towards changing the future of our country *and* earned this badge. By registering to vote, you’re basically the G.O.A.T',
-    unearnedText: (
-      <span>
-        Unlock this badge by taking 2 minutes to{' '}
-        <a
-          href="https://vote.dosomething.org/?r=campaignID:8017,campaignRunID:8022,source:web,source_details:badges"
```
Comment (aaronschachter) [inline resources/assets/components/pages/AccountPage/Badges/BadgesTab.js:136]: 🙀 (part 2 DRY)

**Review by aaronschachter — APPROVED** [review]

File: resources/assets/components/pages/AccountPage/Badges/BadgesTab.js (hunk 3 above)
Comment (lindsayrmaher) [reply resources/assets/components/pages/AccountPage/Badges/BadgesTab.js:136]: 😂  i didn't even put that together. Alas!!

File: docs/development/features/badges.md (hunk 1 above)
Comment (lindsayrmaher) [reply docs/development/features/badges.md:32]: ➕  yeah that makes sense to me. will update!

**Pull request merged**